schedd.go

```diff
@@ -154,6 +192,12 @@ func (s *Schedd) queryWithAuth(ctx context.Context, constraint string, projectio
 	// Receive response ads
 	var jobAds []*classad.ClassAd
 
+	// Determine the limit to apply
+	limit := -1 // unlimited by default
```

Comment:
If a page token is provided, then we need to alter the query to select jobs from after the page token.
- Decode as base64
- Parse as a Cluster.Proc-style identifier.
- Assume remaining jobs come after those jobs (if cluster ID is the same, ProcID must be larger; otherwise, cluster ID must be larger).

This uses the fact we assume the schedd returns job in cluster.proc order.